clients/python-wrapper/lakefs/client.py

```diff
@@ -106,6 +106,17 @@ def version(self) -> str:
         return self._server_conf.version
 
 
+def from_web_identity(code: str, state: str, redirect_uri: str, ttl_seconds: int = 3600, **kwargs) -> Client:
+    """
```

Comment:
Our docstrings generate our documentation. Please add verbose documentation and document parameters as well